sql/pg_duckdb--0.3.0--0.4.0.sql

```diff
@@ -224,6 +224,42 @@ SET search_path = pg_catalog, pg_temp
 AS 'MODULE_PATHNAME', 'duckdb_only_function'
 LANGUAGE C;
 
+CREATE FUNCTION @extschema@.time_bucket(bucket_width interval, ts date)
+RETURNS date
+SET search_path = pg_catalog, pg_temp
+AS 'MODULE_PATHNAME', 'duckdb_only_function'
+LANGUAGE C;
+
+CREATE FUNCTION @extschema@.time_bucket(bucket_width interval, ts timestamp)
```

Comment:
I looked at the result from:
```sql
select return_type, parameters, parameter_types from duckdb_functions() where function_name = 'time_bucket';
```
And the `timestamptz` versions of all these function are missing.